README.md

```diff
@@ -25,18 +25,24 @@
 
 ## Overview
 
+neo-python is an alternative implementation of the original NEO environment. It
```

Comment:
I don't think this paragraph really helps. Perhaps remove it and just add "Build, deploy and run smart contracts" to the bullet list below?